presto-main/src/main/java/com/facebook/presto/sql/planner/EqualityInference.java

```diff
@@ -62,7 +62,7 @@ public int compare(Expression expression1, Expression expression2)
             // Current cost heuristic:
             // 1) Prefer fewer input symbols
             // 2) Prefer smaller expression trees
-            // 3) Ordering.arbitrary() - creates a stable consistent ordering (extremely useful for unit testing)
+            // 3) Sort the expressions alphabetically - creates a stable consistent ordering (extremely useful for unit testing)
```

Comment:
It seems that eeaf2e5 was merged first and I rebased on top of that. I'll make that a 'fix outdated comment' commit.